cli/gitcommands_test.go

```diff
@@ -0,0 +1,49 @@
+package cli
+
+import (
+	"reflect"
+	"testing"
+)
+
+func TestGetRepositoriesList(t *testing.T) {
```

Comment:
I think we can add the following test cases:
1. "repo1;"
I guess the expected output should be: ["repo1"]

2. ";"
I guess the expected output should be: []